examples/lb-provider-reactive/reactive/charm.py

```diff
@@ -1,9 +1,16 @@
 #!/usr/bin/env python3
 
 from charms.reactive import when, when_not, set_flag, endpoint_from_name
+from charms.reactive import hook
 from charms import layer
 
 
+@hook("upgrade-charm")
+def upgrade_charm():
+    lb_consumers = endpoint_from_name("lb-consumers")
+    lb_consumers.follower_perms(read=True)
```

Comment:
I think this would only enable the perms on upgrade-charm; what about new installs? I would consider setting it in a `@hook("install")` handler too, or maybe set the configuration immediately prior to accessing `all_requests`.